### ISSUE TITLE
username and password in Kapacitor for sending messages to a Kafka cluster

### ISSUE THREAD

**Issue by docmerlin** [influxdata/kapacitor#2675]
Feature Request Summary:
 to be able to use a username and password in Kapacitor for sending messages to a Kafka cluster.

Current Kafka output settings:
```
[[kafka]]
  enabled = true
  id = "localhost"
  brokers = []
  timeout = "10s"
  batch-size = 100
  batch-timeout = "1s"
  use-ssl = false
  ssl-ca = ""
  ssl-cert = ""
  ssl-key = ""
  insecure-skip-verify = false
```
telegraf can already do this

**Comment by timhallinflux** [comment]
I think we want to emulate these config settings:

see:
https://github.com/influxdata/telegraf/blob/release-1.21/plugins/inputs/kafka_consumer/README.md
https://github.com/influxdata/telegraf/blob/release-1.21/plugins/outputs/kafka/README.md

  ## with TLS encryption enabled
  # sasl_username = "kafka"
  # sasl_password = "secret"

  ## Optional SASL:
  ## one of: OAUTHBEARER, PLAIN, SCRAM-SHA-256, SCRAM-SHA-512, GSSAPI
  ## (defaults to PLAIN)
  # sasl_mechanism = ""

  ## used if sasl_mechanism is GSSAPI (experimental)
  # sasl_gssapi_service_name = ""
  # ## One of: KRB5_USER_AUTH and KRB5_KEYTAB_AUTH
  # sasl_gssapi_auth_type = "KRB5_USER_AUTH"
  # sasl_gssapi_kerberos_config_path = "/"
  # sasl_gssapi_realm = "realm"
  # sasl_gssapi_key_tab_path = ""
  # sasl_gssapi_disable_pafxfast = false

  ## used if sasl_mechanism is OAUTHBEARER (experimental)
  # sasl_access_token = ""

  ## SASL protocol version.  When connecting to Azure EventHub set to 0.
  # sasl_version = 1```

**Comment by docmerlin** [comment]
https://github.com/influxdata/kapacitor/pull/2677

**Comment by docmerlin** [comment]
closed by #2677
